tests/pkg-rendition-paginated/OPF/package.opf

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" version="3.0" unique-identifier="pub-id" prefix="rendition: http://www.idpf.org/vocab/rendition/#">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Package Rendering Metadata</dc:coverage>
+    <dc:creator>Matt Garrish</dc:creator>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2021-11-18</dc:date>
+    <dc:description>Tests on handling overflow content with the "rendition:flow" property set to "paginated". The content should be dynamically paginated by the reading system.</dc:description>
+    <dc:identifier id="pub-id">pkg-rendition-paginated</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Rendition set to paginated</dc:title>
```

Comment:
Like https://w3c.github.io/epub-tests/#sec-fixed-layout-data, I guess use a title like "rendition:flow paginated results in discrete pages". At least we should say rendition:flow, since there are several rendition properties.

Reply:
I made a bit simpler: replaced "Rendition" to "rendition:flow" (here and in the other two).